app/server/nextkit.ts

```diff
@@ -0,0 +1,10 @@
+import createAPI from "nextkit";
+
+export const api = createAPI({
+  async onError() {
+    return {
+      status: 500,
+      message: "Something went wrong.",
+    };
+  },
+});
```

Comment:
Cool, much cleaner!

Reply:
Most importantly, we get types client-side 🤤